### ISSUE TITLE
Players managed to kill wither pet, which dropped a nether star

### ISSUE THREAD

**Issue by s1h4d0w** [iSach/UltraCosmetics#439]
### What steps will reproduce the issue?  
1. Player spawns the Wither pet
2. They enter the pvp arena of the server, where pvp is enabled. This is a worldguard region where all player commands are disabled aside of messaging related and /exit, which teleports them out. On entry players are healed, fed and given an absorption effect for 10 seconds using WorldGuardExtraFlags.
3. The Wither pet turns hostile and players can kill it
4. On death the Wither pet drops or can drop a nether star, allowing abuse by players

### What was supposed to happen?
Either the pet not turning hostile, or it at least not dropping any items so players can't abuse the pet to farm nether stars

### What happened? 
It was able to be killed and dropped valuable items

### What version of UltraCosmetics are you using?  
2.4.11

### What Spigot version are you using? Paste the output of /version below.  
CraftBukkit version git-Spigot-79a30d7-acbc348 (MC: 1.12.2) (Implementing API version 1.12.2-R0.1-SNAPSHOT)

### What plugins do you have installed? Paste the output of /plugins below.  
KeepItems, PlaceholderAPI, ArmoredElytra, ConsoleSpamFix, SkinsRestorer, AutoRestart, CoreProtect, ClearLag, ChatControl, Genie, ProtocolLib, Multiverse-Core, LibsDisguises, FastAsyncWorldEdit, commandDelay, PermissionsEx, AntiBuild, ServerListPlus, Vault, PlayerMarkers, UltraCosmetics, Matrix, PerWorldInventory, HolographicDisplays, HeadsPluginAPI, PlayerEmotesPro, MyCommand, WorldEdit, Orebfuscator4, NoSwear, AdvancementMoney, Essentials, AutoMessager, OnTime, WorldEditSelectionVisualizer, TouchscreenHolograms, HeadsInventory, WorldGuard, MobStacker, Shop, BetterRTP, EssentialsSpawn, WorldGuardExtraFlags, SilkSpawners, DenyTP, ChopTree3, GriefPrevention, AuthMe, CrazyEnchantments

### Are any errors related to UltraCosmetics in your console or logs? If so, paste below.  
none

### Any additional information that you would like to provide that may be relevant to the issue?
I now execute `/uc clear %username%` when people enter the arena, but I haven't been able to test if that fixes the issue yet.

**WorldGuard region flags:**
```
join-location: World: world, Coordinates: (1255.5, 70, 608.5), Yaw: -180.30591, Pitch: 0.24881662, 
greeting: Remember to use /exit to leave the arena!, 
console-command-on-entry: [/pex user %username% group add pvp, /effect %username% 22 10 5, /feed %username%], 
keep-inventory: true, 
pvp: ALLOW, 
allowed-cmds: [/login, /r, /vote, /msg, /w, /exit, /discord, /register, /mail], 
deny-message: Use /exit to leave the arena!, 
respawn-location: World: world, Coordinates: (1255.5064505476357, 70, 608.5453103923909), Yaw: -180.30591, Pitch: 0.24881662, 
block-break: DENY, 
keep-exp: true, 
block-place: DENY, 
glide: DENY, 
console-command-on-exit: [/pex user %username% group remove pvp], 
mob-spawning: DENY, 
item-durability: DENY
```

**Player logs:**
```
[01:59:22 INFO]: CraftingLGM4u issued server command: /pvp
[02:00:14 INFO]: CraftingLGM4u withered away
[02:00:14 INFO]: [ClearLag] Warning! Dropped items will be removed in 60 seconds!
[02:00:20 INFO]: CraftingLGM4u issued server command: /back
[02:00:44 INFO]: ‹[VIP] ~Crafting› remember my wither?
[02:00:44 INFO]: [ClearLag] Warning! Dropped items will be removed in 30 seconds!
[02:00:47 INFO]: ‹[VIP] ~Crafting› It turned evil
[02:00:47 INFO]: ‹[VIP] ~Tuvu› yas
[02:00:50 INFO]: ‹[VIP] ~Tuvu› wdym
[02:00:54 INFO]: ‹[VIP] ~Crafting› and attacked me when i wanted to go into the arena
[02:00:58 INFO]: ‹[VIP] ~Crafting› i did /pvp
[02:00:59 INFO]: ‹[VIP] ~Tuvu› LOL
[02:01:00 INFO]: ‹[VIP] ~Crafting› and he was here
[02:01:02 INFO]: ‹[VIP] ~Crafting› I had to kill him
[02:01:02 INFO]: ‹[VIP] ~Tuvu› hahaha
[02:01:05 INFO]: ‹[VIP] ~Tuvu› so u can kill it
[02:01:06 INFO]: ‹[VIP] ~Crafting› Got a nether start tho lol
```

**Comment by iSach** [comment]
Can you try on 2.5?

Sorry for the inconvenience.

**Comment by iSach** [comment]
Should be fixed in 2.5.1
Please re-open if it's not. Sorry again for the inconvenience and delay.

**Comment by s1h4d0w** [comment]
Seems to be fixed on latest version, as I replied to the other thread I've switched to 1.14 and made a quick PVP WorldGuard region, the pet no longer turned hostile.
